app/controllers/cars_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CarsController < ApplicationController
+  before_action :set_car, only: %i[show edit update destroy]
+
+  # GET /cars or /cars.json
+  def index
+    @cars = Car.all
+
+    render json: @cars
+  end
+
+  # POST /cars or /cars.json
+  def create
+    @car = Car.new(car_params)
+    # @car.user_id(current_user.id)
+    if @car.save
+      render json: @car, status: :created, location: @car
+    else
+      render json: @car.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /cars/1 or /cars/1.json
+  def destroy
+    @car.destroy
+
+    render json: { message: 'Car was successfully destroyed.' }
+  end
```

Comment:
- [ ] What happens if the car is not successfully destroyed?  Kindly add some error handling like you did on the create method (line 12). 👍